mm/alignment.py

```diff
@@ -15,6 +16,12 @@
 
 
 class AlignmentJob(rasr.RasrCommand, Job):
+    """
+    Align a dataset with the given feature scorer.
+    """
+
+    __sis_hash_exclude__ = {"plot_alignment_scores": True}
```

Comment:
A very good idea to add plotting to the AlignmentJob, which I would endorse.

I my opinion the `__sis_hash_exclude__` should be with the value that recovers the old behavior. And then also the default value should be the one with the default behavior. Otherwise some jobs that have been run with a previous commit would now have the output but not have the file created.

Reply:
I thought about it, but I didn't consider it an issue. Yes, the functionality (i.e. spawned tasks) changes, but either way the user wasn't expecting to use the plots, so nothing really changed from the point of view of the user, I think.

If you insist on this, I will revert to the default behavior.

Reply:
> If you insist on this

lets say I have a weak preference. But I will be ok if others agree with you

Reply:
Meh, I can live with this. An approval in the hand is worth two in the bush :woman_shrugging: